registry.nuon

```diff
@@ -26,21 +26,166 @@
 
 {
     git: [
-        [name version url revision path];
         [
-            nu-git-manager
-            0.5.0
-            https://github.com/amtoine/nu-git-manager.git
-            0.5.0
+            name,
+            version,
+            url,
+            revision,
+            path
+        ];
+        [
+            nu_plugin_audio_hook,
+            "0.1.12",
+            "https://github.com/FMotalleb/nu_plugin_audio_hook",
+            "0.1.12",
+            .
```

Comment:
I feel like we should allow `null` here.

Reply:
yeah, could allow that as well :+1: